### DISCUSSION TITLE
rebuild-database

### DISCUSSION THREAD

**Discussion by mevitsbricks** [rgriebl/brickstore#437] [Q&A] [answered]
Just wondering what exactly is the command line option --rebuild-database do?

Thanks

**Comment by rgriebl** [comment, marked as answer]
This is for rebuilding the database, obviously 😏 
Just kidding, but it really does re-build the database that BrickStore is downloading every now and then from my server. You shouldn't run this yourself, because it will download *a lot of* data from BL (first run is in the hours).
So, as long as you are not planning to provide the database to other people (like I do), then just ignore the option ... plus, I've removed it from the normal desktop exe in the newest dev builds anyway: only the Debian/backend package is able to rebuild the DB nowadays.

**Reply by mevitsbricks** [reply]
Thanks, I actually ran it and was like, holy cow!

**Comment by Sofrosyn** [comment]
What if I want to provide the database. Is there any documentation on that
